### ISSUE TITLE
`@stylistic/value-list-comma-space-before` tells the reader whitespace is unexpected where the option asks for a space

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#175]
### What minimal example or steps are needed to reproduce the bug?

```css
a { prop: 0,0; }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/value-list-comma-space-before": "always-single-line"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json
```

### Which Stylelint-related dependencies are you using?

```json
{
  "@stylistic/stylelint-plugin": "5.3.0",
  "stylelint": "17.14.1"
}
```

### What did you expect to happen?

The rule asks for a space in front of the comma, so the warning says so:

```
1:12  Expected single space before "," in a single-line list  (@stylistic/value-list-comma-space-before)
```

### What actually happened?

It says the opposite of what the option asks for:

```
1:12  Unexpected whitespace before "," in a single-line list  (@stylistic/value-list-comma-space-before)
```

The reader is told to take whitespace away where the rule wants it put in, and a fix then puts it in, which reads as the fix doing the opposite of what was reported.

### Do you have a proposal to fix the bug?

`messages.expectedBeforeSingleLine` is spelled with the text of `messages.rejectedBeforeSingleLine`:

```js
export let messages = ruleMessages(ruleName, {
	expectedBefore: () => `Expected single space before ","`,
	rejectedBefore: () => `Unexpected whitespace before ","`,
	expectedBeforeSingleLine: () => `Unexpected whitespace before "," in a single-line list`,
	rejectedBeforeSingleLine: () => `Unexpected whitespace before "," in a single-line list`,
})
```

It reads `Expected single space before "," in a single-line list`, which is what `media-query-list-comma-space-before` spells for the same message. This is the only one of its kind in the plugin: no other rule gives an `expected*` message the text of a `rejected*` one.

The tests ask for the message through `messages.expectedBeforeSingleLine()`, so none of them notices, and the text is what a user sees and what a user matches on, so the change belongs in the changelog under `Changed`.
